### ISSUE TITLE
No lava and no vanilla caves generating

### ISSUE THREAD

**Issue by ProjectNextra** [VolmitSoftware/Iris#195]
**Describe the bug**
- Lava is not generaring. In caves, underground or on the surface.

- Vanilla caves are not generating. When both cave types are activated in config only Iris caves generate, when just vanilla is activated only ravines and mineshafts seem to generate, without any cave systems at all.

**Expected behavior**
Lava generating in the world
Vanilla caves generating

Server and Plugin Informations

- Installed plugins:
- Iris version: 1.1.3
- Multiverse Core 4.2.2
- Server Platform and Version: Paper 1.16.4 294
- Server logs: none

**Comment by svdgoor** [comment]
Lava not spawning is not a bug but it would be great if we could add a setting for forcing lava at a certain height in caves.
As for the caves, I confirm this bug.

**Comment by cyberpwnn** [comment]
You can add fluids to iris caves, its called fluid (in a cave layer)

```json
"caveLayers": [
    {
       
        "fluid": {
            "fluidType": {
                "block": "lava"
            },
            "fluidHeight": 8
        }

    }
],
```

I think vanilla caves were disabled once we switched to v2 arch. Ill re-add it if possible.

**Comment by ProjectNextra** [comment]
i see that code is present in overworld.json but there's no lava even if i bump up the fluidHeight. I changed it to 20 to test but still nothing

**Comment by svdgoor** [comment]
@ProjectNextra Can you try bumping it up to 100 and seeing if that still doesn't work, please?
Trying to figure out if this is due to a bug in the plugin or a config error in the pack.

**Comment by svdgoor** [comment]
Others have gotten this to work so changing this to be an Overworld pack enhancement.

**Comment by svdgoor** [comment]
Splitting off vanilla cave issue since it's separate.

**Comment by ghost** [comment]
fixed the lava thing with v1.2.1
